### DISCUSSION TITLE
WRF Hydro standalone Parallel Mode with PGI Compiler

### DISCUSSION THREAD

**Discussion by FatemehAmeri** [NCAR/wrf_hydro_nwm_public#768] [Q&A]
I have compiled WRF Hydro  standalone in parallel mode using the PGI compiler. When I tried to run the tests, both with one processor and with multiple processors (e.g., 8), the simulation time was the same. I came across something in the user manual (2.3 Parallelization Strategy) that says, "Currently, sequential compile is not supported, so MPI libraries are required even if running over a single core."

Does this mean that it does not support multicore processing? I would really appreciate any help or clarification on this issue.

**Comment by scrasmussen** [comment]
Discussion is repeat of issue #767 and has been answered. Closing discussion unless there are follow-up questions.
